### PR TITLE
Public demoshop update

### PR DESCRIPTION
**Pull request by TomasGottvald**
| Q             | A
| ------------- | ---
|Description, reason for the PR| change blue to gold color, change contact person, add copyright image info, change footer text, remove unneded advert bottom gap, product list - bigger image, hide description to show on our public demoshop
|New feature| No 
|Fixes issues| #60, #54, #49 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

User logged menu before:
https://user-images.githubusercontent.com/10008612/58613302-9f035d00-82b5-11e9-858d-1a1691666693.png

User logged menu after:
obrazek

Registration error message before:
https://user-images.githubusercontent.com/1177414/61213783-53423100-a706-11e9-994b-5b6e8b2bf433.png

Registration error message after:
obrazek

### CODE REVIEW

**Review by LukasHeinz — CHANGES_REQUESTED** [review]
It looks good.

But the change of color to gold is not done through all domains on 2. and 3. domain there is mixture of gold and green color. Either use green or gold. Not together.
Snímek obrazovky 2019-09-06 v 14 59 50

**Comment by TomasGottvald** [comment]
I just resolved issues and made color changes for another domains.

**Review by LukasHeinz — APPROVED** [review]

File: src/Shopsys/ShopBundle/Resources/views/Front/Layout/footer.html.twig [hunk 1]
```diff
@@ -9,10 +9,19 @@
...
+                <a href="https://www.artsperky.cz" target="_blank">artsperky.cz</a>.
+                {{ 'Do not distribute without permission.'|trans }}
+            </span>
+        </div>
+    </div>
     <div class="web__line">
         <footer class="web__container footer__bottom">
             <div class="footer__bottom__copyright dont-print">
-                Made with passion by <a href="https://www.shopsys.com/">Shopsys Framework</a>
+                {{ 'Powered by'|trans }} <a href="http://www.shopsys-framework.com/">Shopsys Framework</a>
```
Comment (DavidOstrozlik) [inline src/Shopsys/ShopBundle/Resources/views/Front/Layout/footer.html.twig]: Why should we use URL that is only http and redirects to the same place that the old one?

```suggestion
                {{ 'Powered by'|trans }} <a href="https://www.shopsys.com/">Shopsys Framework</a>
```

**Review by DavidOstrozlik — CHANGES_REQUESTED** [review]
Hi. I added one suggestion please take a look at it.

**Comment by TomasGottvald** [comment]
> 
> 
> Hi. I added one suggestion please take a look at it.

Yes it makes sense. My update was according to Kamil´s notes.

**Review by LukasHeinz — CHANGES_REQUESTED** [review]
It is grat, only the FE user manu has have nicer dropdown - the curernt does not match any stly in FE and doesnt fit the menu.
Snímek obrazovky 2019-09-10 v 9 24 21

**Comment by TomasGottvald** [comment]
After discussion we agreed on these issues:
- cookie button - change color to gold
- active category color - change to gold
- change blue color on mobile menu
- change logged user menu - 13px font, 10px padding, remove hover background effect

**Review by LukasHeinz — DISMISSED** [review]
On 2nd and 3rd domain there is still secondary color in category menu.
Snímek obrazovky 2019-09-10 v 12 55 09
Snímek obrazovky 2019-09-10 v 12 58 42

**Comment by TomasGottvald** [comment]
@PetraPiprekova fixed #54 - fixed for german language.

**Review by PetraPiprekova — APPROVED** [review]
tests

**Review by DavidOstrozlik — APPROVED** [review]

**Pull request merged**